src/app/portlets/content-types/main/content-types.component.ts

```diff
@@ -49,72 +57,101 @@ export class ContentTypesPortletComponent extends BaseComponent {
+                        command: $event => {
                             this.createContentType('file', $event);
                         },
                         icon: 'fa-file-o',
                         label: this.i18nMessages['File']
                     },
                     {
-                        command: ($event) => {
+                        command: $event => {
                             this.createContentType('page', $event);
                         },
                         icon: 'fa-file-text-o',
                         label: this.i18nMessages['Page']
                     },
                     {
-                        command: ($event) => {
+                        command: $event => {
                             this.createContentType('persona', $event);
                         },
                         icon: 'fa-user',
                         label: this.i18nMessages['Persona']
-                    }]
+                    }
+                ]
             },
             secondary: [
                 {
                     label: this.i18nMessages['Actions'],
                     model: [
                         {
-                            command: ($event) => {
+                            command: $event => {
                                 // DELETE DATA
                             },
                             deleteOptions: {
-                                confirmHeader: this.i18nMessages['message.structure.cantdelete'],
-                                confirmMessage: this.i18nMessages['message.structure.delete.structure.and.content'],
+                                confirmHeader: this.i18nMessages[
+                                    'message.structure.cantdelete'
+                                ],
+                                confirmMessage: this.i18nMessages[
+                                    'message.structure.delete.structure.and.content'
+                                ]
                             },
                             icon: 'fa-close',
                             label: this.i18nMessages['delete']
-                        },
+                        }
                     ]
                 }
             ]
         };
 
         this.contentTypeColumns = [
-            {fieldName: 'name', header: this.i18nMessages['Structure-Name'], width: '35%', sortable: true},
-            {fieldName: 'velocityVarName', header: this.i18nMessages['Variable'], width: '10%'},
-            {fieldName: 'description', header: this.i18nMessages['Description'], width: '35%'},
-            {fieldName: 'nEntries', header: this.i18nMessages['Entries'], width: '10%'},
-            {fieldName: 'mod_date', header: 'Last Edit Date', width: '10%', format: 'date', sortable: true}
+            {
+                fieldName: 'name',
+                header: this.i18nMessages['Structure-Name'],
+                icon: (item: any): string => this.contentTypesInfoService.getIcon(item.type),
```

Comment:
Can't do... item is way different than 'ContentType', the endpoint return different things when you do listing or get by id.